test/utils/runners.go

```diff
@@ -321,7 +322,8 @@ func (config *DeploymentConfig) create() error {
 					Annotations: config.Annotations,
 				},
 				Spec: v1.PodSpec{
-					Affinity: config.Affinity,
+					Affinity:                      config.Affinity,
+					TerminationGracePeriodSeconds: func(i int) *int64 { x := int64(math.Max(float64(i), 1)); return &x }(config.TerminationGracePeriodSeconds),
```

Comment:
i think it's better to not use an anonymous function and instead define it in the runners.go level for all create() methods to use.

Reply:
Shouldn't the default be whatever the k8s default is and not 1?
Why do we even need any function (other than Max)? Maybe we could use https://github.com/kubernetes/utils/blob/master/integer/integer.go#L20?

Reply:
@neolit123 Thanks for the quick review. Made the change.

@MaciekPytel The default that was being used for RCConfig was 1, so kept that.
The file already uses math.Max for ints so kept that too.